fuels-contract/src/contract.rs

```diff
@@ -159,7 +148,35 @@ impl Contract {
         if let Some(e) = encoded_args {
             script_data.extend(e)
         }
+        Ok((script, script_data))
+    }
 
+    /// Calls an already-deployed contract code.
```

Comment:
```suggestion
    /// Calls a contract method.
```

Just a nit; I don't see when one would call a non-deployed contract method.